static/app/views/replays/details.tsx

```diff
@@ -156,8 +156,4 @@ const Side = styled(Layout.Side)`
   padding-bottom: ${space(2)};
 `;
 
-const BreadcrumbTimeline = styled(ReplayBreadcrumbOverview)`
-  max-height: 5em;
-`;
```

Comment:
This wasn't doing anything, the height is under the max, and 5 was an arbitrary number anyway.